plugins/auth-backend/src/lib/oauth/OAuthAdapter.ts

```diff
@@ -246,6 +254,27 @@ export class OAuthAdapter implements AuthProviderRouteHandlers {
     return req.cookies[`${providerId}-scope`];
   };
 
+  private setAccessTokenCookie = (
```

Comment:
`AccessToken` is very misleading. We are not creating a new cookie for accessing resources, so my recommendation would be `access` or `authentication` cookie, just so that it is not misleading.